devfilev2.yaml

```
@@ -0,0 +1,77 @@
+schemaVersion: 2.1.0
+metadata:
+  generateName: che-docs
+components:
+  - attributes:
+      app.kubernetes.io/name: che-docs
+      che-theia.eclipse.org/vscode-extensions:
+        - "https://github.com/golang/vscode-go/releases/download/v0.23.0/go-0.23.0.vsix"
+        - "https://github.com/microsoft/vscode-python/releases/download/2020.7.94776/ms-python-release.vsix"
+        - "https://github.com/redhat-developer/codeready-workspaces-vscode-extensions/releases/download/7.44-che-assets/java-0.82.0-369.vsix"
+        - "https://github.com/redhat-developer/codeready-workspaces-vscode-extensions/releases/download/v069c634/vscode-xml-0.17.0.vsix"
+        - "https://open-vsx.org/api/redhat/vscode-yaml/0.14.0/file/redhat.vscode-yaml-0.14.0.vsix"
+        - "https://open-vsx.org/api/vscode/typescript-language-features/1.49.3/file/vscode.typescript-language-features-1.49.3.vsix"
+        - "https://github.com/redhat-developer/codeready-workspaces-vscode-extensions/releases/download/7.44-che-assets/vscode-java-debug-0.26.0.vsix"
+        - "https://open-vsx.org/api/vscjava/vscode-java-test/0.28.1/file/vscjava.vscode-java-test-0.28.1.vsix"
+      che-theia.eclipse.org/vscode-preferences:
+        java.server.launchMode: Standard
+    container:
```

Is it possible to move these attributes to another configuration file?

```suggestion
  - container:
```